wasmcontext.go

```diff
@@ -34,72 +34,74 @@ func (i *Instance) compile(wasmbytes []byte) {
 	// XQD Stubbing -{{{
 	// TODO: All of these XQD methods are stubbed. As they are implemented, they'll be removed from
```

Comment:
these are no longer xqd, right?

Reply:
That's correct. They've all be renamed into `fastly_xyz` modules. I stopped short of doing more renames to clean that all up. 

See: https://docs.rs/fastly-sys/0.4.0/fastly_sys/index.html

Reply:
I just mean that the TODO comment is now confusing, or I might just not understand it

Reply:
🤦  Ah yes.  I'll fix up some comments/docs.